lib/algos/list/delimiterMaster.js

```diff
@@ -156,9 +156,10 @@ class DelimiterMaster extends Delimiter {
      *  @return {number}          - indicates if iteration should continue
      */
     filterV1(obj) {
-        // Filtering master keys in v1 is simply listing the master
-        // keys, as the state of version keys do not change the
-        // result, so we can use Delimiter method directly.
+        const value = obj.value;
```

Comment:
we should keep the comment & "improve it" to document what you change : because indeed (as was indicated), filtering in v1 is not really needed..........expect we need it to handle the case of objects deleted while a bucket was version-suspended